src/test/scala/memeid/UUIDSpec.scala

```diff
@@ -146,3 +156,48 @@ class UUIDSpec extends Specification with ScalaCheck {
   }
 
 }
+
+class V1Spec extends Specification with ScalaCheck {
```

Comment:
I would move version specs to its own files to avoid having super-big test files